java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/build/preprocessor/AbstractPreprocessor.java

```diff
@@ -113,6 +113,15 @@ public void run() {
                 PATH="${JAVA_HOME}/bin:$PATH"
             fi
 
+            # Go through certificates and insert them into the cacerts
+            for cert in /etc/pki/ca-trust/source/anchors/*; do
+              echo "inserting $cert into java cacerts"
+              keytool -import -alias $(basename $cert)-ca \
+                  -file $cert \
+                  -keystore /etc/pki/java/cacerts \
+                  -storepass changeit --noprompt
+            done;
+
             if [ -n "${MAVEN_HOME}" ]; then
```

Comment:
I'm embedding a slightly modified version of this in https://github.com/redhat-appstudio/jvm-build-service/pull/2286/files#diff-181fac686bf877763267bd8f7f73c8b1c471c4b6cde00a636da1ba911d4bb233R150